openbas-model/src/main/java/io/openbas/database/model/Channel.java

```diff
@@ -93,6 +93,10 @@ public class Channel implements Base {
   @Transient
   private final ResourceType resourceType = ResourceType.CHANNEL;
 
+  @OneToMany(mappedBy = "channel", fetch = FetchType.LAZY)
+  @JsonIgnore
+  private List<Article> articles;
```

Comment:
Private field 'articles' is assigned but never accessed

Reply:
This is required for JPA queries like findAllDistinctByScenarioId. If the field didn't exist, I would have had to use a native query instead. I can still do that if you prefer.